maxim/logger/smolagents/client.py

```
@@ -0,0 +1,539 @@
+            "total_tokens": total_tokens,
+        },
+    }
+    
+    # Add additional metadata if available
+    if hasattr(agent, 'tools') and agent.tools:
+        result_data["tools_used"] = [tool.name if hasattr(tool, 'name') else str(tool) for tool in agent.tools]
+    
+    generation.result(to_serializable_dict(result_data))
+
+
+def instrument_smolagents_tools():
```

Ruff (0.12.2)

Missing return type annotation for public function `instrument_smolagents_tools`

Add return type annotation: `None`

(ANN201)